docs/doc/30-reference/20-functions/20-numeric-functions/ceil.md

```diff
@@ -21,17 +19,17 @@ CEIL(x)
 
 ## Return Type
 
-A Float64 data type value.
+A numeric data type value which is the same as input type.
```

Comment:
What database is this behavior aligned to? As far as I know, mysql and postgresql return type is an integer

Reply:
I referred to duckdb and clickhouse.

PG returns numeric type too.  

https://www.db-fiddle.com/f/oAKgKRCktppWXb31Zw3Sws/0